test/stake/StakingRewardsCampaign.ts

```diff
@@ -0,0 +1,249 @@
+import {expect} from "chai";
+import {ethers, upgrades} from "hardhat";
+import {BigNumber, Contract, ContractFactory} from "ethers";
+import {SignerWithAddress} from "@nomiclabs/hardhat-ethers/signers";
+import { days } from "@nomicfoundation/hardhat-network-helpers/dist/src/helpers/time/duration";
+import { formatEther, parseEther } from "ethers/lib/utils";
+import { time } from "@nomicfoundation/hardhat-network-helpers";
+
+const getDays = (day: number) => {
+    return 24 * 60 * 60 * day;
+}
+
+const tier = ethers.utils.formatBytes32String("LUCKY_DRAW.TIER_1")
```

Comment:
Please add a test that tries to verify with another "tier":

Test if staking into LUCKY_DRAW.TIER_2 is possible, and does not affect LUCKY_DRAW.TIER_1